metaflow/plugins/airflow/plumbing/set_parameters.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+import sys
+
+
+def export_parameters(output_file):
+    input = json.loads(os.environ.get("METAFLOW_PARAMETERS", "{}"))
```

Comment:
Do we need to handle parameters this way for Airflow? This specific approach was a workaround for all the limitations Step Functions was imposing on us. Perhaps there is a cleaner, more Airflow-native way of passing parameters?

Reply:
Can you expand on what you mean by a more airflow native way? I use the `{{params}}` macro from airflow to pass the parameters to an environment variable. These are then used to set `METAFLOW_INIT_` env variable.